### ISSUE TITLE
Some new ideas (convert to PDF & add different languages)

### ISSUE THREAD

**Issue by Marie-P** [mamamia5x/markdownpedia#29]
:sun_with_face:  Hello :sun_with_face: 

We are three Computer Sciences :computer: students and want to contribute on Markdownpedia.

Actually, we have two ideas :bulb: to contribute the project.
In wikipedia we can download as pdf and have different languages. We would like to create an article, as an example, in a few languages. So, add buttons to change language and convert the page to PDF. 
:thought_balloon: Is it interesting for markdownpedia ? 

:sparkles: Best regards.

**Comment by phv4ng** [comment]
On **Wikipedia** there are many options to share the article, by languages or to download the page to keep it offline.

# Create the first article 1️⃣ 
To begin with, a first article's creation will be use as an example. 
1. To build the style sheet around the article's page
2. To fill the **Markdowpedia**, currently's empty

After that, buttons need to be added to change the language's page and to convert the page as PDF :

## 1. Transcript the article in different languages

## 2. Convert Pages as PDF

**Comment by mamamia5x** [comment]
Yeah, it'll be an awesome idea. I myself have to fix the site, for some reason, the hosting service doesn't want to work. 
You guys can go right ahead, these will be great additions to the site, and thank you for choosing Markdownpedia.

**Comment by mamamia5x** [comment]
The site is back in action, I'll be doing some fixes to the backend. 
But this does have me thinking. 

Markdownpedia gets markdown files and then converts them into HTML files and sends it to the user. Would it be possible to have another script convert the html page to a pdf file and have a button that sends that file to the user?

What do you guys think? There is jsPDF, which seems more like writing on a canvas.

**Comment by aboughatane** [comment]
Hello, 
Yeah, we think it's a good idea, we'll do it like that.

**Comment by mamamia5x** [comment]
Alright sounds good, would you like me to assign you?

**Comment by Marie-P** [comment]
Yes it would be great.

**Comment by aboughatane** [comment]
Hello, 
Sorry to come back this late :confused:, we were too busy for school. We did a pull request, tell us what you think about it.



![pdfConvert](https://user-images.githubusercontent.com/52924538/117535685-2fe12800-aff7-11eb-8474-745b5820e871.png)
